lib/govuk_template/version.rb

```diff
@@ -1,3 +1,3 @@
 module GovukTemplate
-  VERSION = "0.21.0"
+  VERSION = "0.21.1"
```

Comment:
I was so focussed on making sure I didn't miss the CHANGELOG entry from the other PR I didn't do the obvious thing!  Thanks for checking.  I've fixed it in a rebase.